### ISSUE TITLE
Unable to install uBlock Origin - "TB cannot modify needed file"

### ISSUE THREAD

**Issue by wonbug** [rusty-snake/firejailed-tor-browser#36]
Using Debian 11 testing, installed firejail from source and the latest version of Tor Browser.

When trying to install uBlock Origin, addons.mozilla.org errors:

    could not be installed because Tor Browser cannot modify the needed file

Oddly, uBlock Origin shows up in the addons list after this, but can't be interacted with and there's no icon in the toolbar.

**Comment by rusty-snake** [comment]
`noblacklist /tmp` seems to work, can you confirm.

---

> **It's strongly discouraged to install new add-ons in Tor Browser, because they can compromise your privacy and security.**

https://support.torproject.org/tbb/tbb-14/

**Comment by wonbug** [comment]
Thanks, that unblocks the download from starting and it appears as though uBlock Origin gets installed, but still no icon appears and I can't access the extension by direct URL either.

I'm aware installing add-ons is discouraged and it's an acceptable risk tradeoff for me.

**Comment by rusty-snake** [comment]
1. So the "… could not be installed because Tor Browser cannot modify the needed file" is caused by `blacklist /tmp` in firejailed-tor-browser.profile. I'm not going to change this in the profile as you should not install Add-Ons in the Tor Browser anyway.
2. The "installed uB does not work, no icon" issue can be reproduced in a fresh installed tor browser (security-level=standard, no FTB). Seems to be out of scope here. You can try to report this to uB/TB.

**Comment by rusty-snake** [comment]
FYI

https://www.reddit.com/r/uBlockOrigin/comments/sdfgzx/no_icon_visible_in_tor_browser/

It seems to work if you `Run in Private Windows = Allow`.

**Comment by wonbug** [comment]
Thanks again for your help and the awesome open source project @rusty-snake
